### ISSUE TITLE
[Bug Report] v-list has a different font when not a direct child of v-navigation-drawer

### ISSUE THREAD

**Issue by ScreamZ** [vuetifyjs/vuetify#5430]
### Versions and Environment
**Vuetify:** 1.3.3
**Vue:** 2.5.17
**Browsers:** Chrome 69.0.3497.100
**OS:** Mac OS 10.14.0

## Introduction

Hello,

I'm not sure this is a « bug » I just might be wrong and missing something but shouldn't styles beings propagated ?

Thanks for your future answer,
Best regards

### Expected Behavior
I would like to have the same styles as it would be all of this in one component.

### Actual Behavior
Some class are not applied when splitting between multiple components

### Reproduction Link
[![Edit Vue Template](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/4j8j7zw4ox?module=%2Fsrc%2FApp.vue)

**Comment by ScreamZ** [comment]
Humm… At last news there is no thing like `React's Fragment`in Vuejs and I need something to wrap.

I switched to this, which wrap all with a `v-list` it seems to work…
Thanks for the feedback @jacekkarczmarczyk.

```html
 <v-navigation-drawer v-model="showDrawer" fixed app>
      <v-list>
        <NavigationDrawer :user="user" @logout="logout"/>
      </v-list>
</v-navigation-drawer>
```

I'm wondering how we could improve this use case ? Any ideas ?

Have a good afternoon :)

**Comment by ScreamZ** [comment]
@KaelWD Nice catch! 

I'll complete your answer, for `v-list` this has to be the following ` <v-list class="py-0"></v-list>` or a padding is applied by default deforming the interface.

I didn't knew about multiple root and functional components, nice

Thanks for the reply.

**Comment by johnleider** [comment]
This is resolved for v2.0 where you must specify **dense** on `v-list`.
